pkg/builder/docker/builder.go

```diff
@@ -32,9 +32,16 @@ func (b *Builder) Build(ctx context.Context, app *builder.AppContext, out chan<-
 	msgc := make(chan string)
 	errc := make(chan error)
 	go func() {
+		args := make(map[string]*string)
+		for k := range app.Ctx.Env.ImageBuildArgs {
+			v := app.Ctx.Env.ImageBuildArgs[k]
```

Comment:
I think this can be removed if we use the form `for k, v := range app.Ctx.Env.ImageBuildArgs {`. Would that work?